CHANGES.md

```diff
@@ -1,3 +1,21 @@
+## v0.1.2
```

Comment:
Were there any data plane changes of note?

Reply:
/cc @olix0r @carllerche @seanmonstar

Reply:
Nothing from me

Reply:
No other changes.